bundles/org.openhab.core.io.net/src/main/java/org/openhab/core/io/net/exec/ExecUtil.java

```diff
@@ -85,9 +85,10 @@ public static String executeCommandLineAndWaitResponse(String commandLine, int t
                 logger.debug("exit code '{}', result '{}'", exitCode, result);
                 return result.toString();
             } catch (IOException e) {
-                logger.warn("I/O exception occurred when executing commandLine '{}'", commandLine, e);
+                logger.debug("I/O exception occurred when executing commandLine '{}'", commandLine, e);
             } catch (InterruptedException e) {
-                logger.warn("Timeout occurred when executing commandLine '{}'", commandLine, e);
+                logger.warn("Timeout occurred when executing commandLine '{}'", commandLine);
+                logger.debug("{}", e);
```

Comment:
The stacktrace should not be substituted into the log message:

```
[ERROR] org.openhab.core.io.net.exec.ExecUtil.java:[91]
Count of placeholder(1) is not equal to count of parameter(0)
```

Reply:
Oops, thanks - fixed!